monai/transforms/spatial/array.py

```diff
@@ -3265,6 +3270,9 @@ class RandGridPatch(GridPatch, RandomizableTransform):
         pad_mode: refer to NumpyPadMode and PytorchPadMode. If None, no padding will be applied. Defaults to ``"constant"``.
         pad_kwargs: other arguments for the `np.pad` or `torch.pad` function.
 
+    Returns:
+        MetaTensor: A MetaTensor consisting of a batch of all the patches with associated metadata
```

Comment:
The same comments with above.

Reply:
Hi @KumoLiu, what do you mean here?

Reply:
Hi @drbeh, sorry for my misunderstood, just ignore this comment.